src/component/AddItem.js

```diff
@@ -4,9 +4,21 @@ import { withFirestore } from "react-firestore";
 const AddItem = ({ firestore }) => {
   const [name, setName] = useState("");
 
-  //   Write item to Firebase
+  //   Write item to Firebase setting uniqueToken as document name
+
+  const uniqueToken = localStorage.getItem("uniqueToken");
+
   const addItem = name => {
-    firestore.collection("items").add({ name });
+    firestore
+      .collection("lists")
+      .doc(uniqueToken)
+      .set({ items: "" });
```

Comment:
figuring out the difference between add and set here was crucial it seems. great job!!